Docs/add 3dp case (AIV-696)

kaitkj opened the pull request.

wujiangang commented on docs/en/get-started/ESP32-S3-EYE_Getting_Started_Guide.md:
```diff
@@ -339,4 +341,22 @@ In case this issue occurs, you need to:
+
+
+
+
```
Remove multiple empty lines to clean up the document.

wujiangang commented:
LGTM. Please squash the commits into a single one in the final stage.

kaitkj commented:
> LGTM. Please squash the commits into a single one in the final stage.

Fixed typo in formating, removed spacing, added URL for Bambulab P1S for both EN and CN.

sun-xiangyu commented:
@kaitkj , @wujiangang , thanks for your PR, I will merge it.
